charts/traccar/values.yaml

```diff
@@ -115,18 +120,14 @@ mysql:
     persistence:
       enabled: false
 
-# NOTE: When using multiple replicas, you must configure broadcast.type and broadcast.address
+# NOTE: When using multiple replicas, you must configure broadcast.enabled
```

Comment:
In my opinion you should design the whole chart differently and use only the environment variables and not the conf generation in the configmap, this would simplify everything.

Unfortunately, I don't have the time to review the whole chart and try to eliminate the whole thing.